velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -814,4 +814,14 @@ struct MillisToTimestampFunction {
   }
 };
 
+template <typename TExec>
+struct SecondsToTimestampFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(TExec);
+
+  template <typename T>
+  FOLLY_ALWAYS_INLINE void call(out_type<Timestamp>& result, const T& seconds) {
```

Comment:
const T& -> T as T is fixed width type and we can pass by value.

Reply:
Thanks @rui-mo for this advice. But could you please explain more details about why we prefer T for fixed type. At first, I choose const T& to try to align with all other timestamp related functions, like [MillisToTimestampFunction](https://github.com/facebookincubator/velox/blob/a0222abbafd78678a3a3fed550d1765230374c84/velox/functions/sparksql/DateTimeFunctions.h#L812)

Reply:
It is a minor suggestion to simplify, because I assume in general, for a double or float variable, it's usually fine to pass by value, as its size is not large enough to cause significant overhead. Thanks.

Reply:
Yeah, I agree that for most of these cases, passing value is enough. However, I suppose here maybe it's better to keep the same style by using const T&.